plugins/modules/aci_l4l7_device.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l4l7_device
+short_description: Manage L4-L7 Devices (vns:LDevVip, vns:RsALDevToPhysDomP)
+description:
+- Manage L4-L7 Devices.
+options:
```

Comment:
document default APIC behaviour when not provided in descriptions

Reply:
updated descriptions